package.json

```
@@ -0,0 +1,68 @@
+{
+  "name": "jest-elasticsearch",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.0.0",
```

1.0.0 will come a bit later)